### DISCUSSION TITLE
Preset group:pulumi is grouping runtime from ASDF

### DISCUSSION THREAD

**Discussion by 1oglop1** [renovatebot/renovate#25182] [Request Help] [answered]
### How are you running Renovate?

Placeholder value, please select the correct response from the dropdown

### If you're self-hosting Renovate, tell us what version of Renovate you run.

gitlab v16.11.0

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### Was this something which used to work for you, and then stopped?

Placeholder value, please select the correct response from the dropdown

### Describe the problem

I asdf to install pulumi and keep it up to date because libraries are useless without the runtime. 

I wanted to use the preset `group:pulumi` to update everything at once.
But it wasn't grouping asdf dependency in the same PR.

Here's my config

```json5
{
  $schema: "https://docs.renovatebot.com/renovate-schema.json",
  extends: ["local>mygroup/renovate-runner//presets/base.json5"],

  ignoreScripts: false,
  packageRules: [
    {
      automerge: true,
      schedule: ["at any time"],
      extends: ["packages:linters", "packages:test"],
      matchPackageNames: ["lerna", "ts-essentials", "pnpm", "@trivago/prettier-plugin-sort-imports"],
      matchPackagePrefixes: ["@types/"],
    },

  ],
  rangeStrategy: "pin",
}
```

`mygroup/renovate-runner//presets/base.json5`
```
{
  extends: [
    "config:recommended",
    "group:pulumi",
  ],
  packageRules: [
    {
      groupName: "Pulumi",
      rangeStrategy: "pin",
      schedule: ["at any time"],
    },
  ],
}
```

I would like to achieve that any pulumi library is grouped but when pulumi runtime is updated it would group it as well.
Eg. for NodJS I did this to get the result.

```
{
  packageRules: [
    {
      matchPackageNames: [
        "pulumi/pulumi",  // this is from asdf
        "@pulumi/pulumi",
      ],
      groupName: "Pulumi",
      rangeStrategy: "pin",
      schedule: ["at any time"],
    },
  ],
}
```

But I am not sure this works for all supported languages.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

Placeholder value, please select the correct response from the dropdown

**Comment by rarkins** [comment, marked as answer]
You can write a package rule to add the missing one to the group. Just make sure to use the same `groupName` as the others have
